packages/api/tests/lambdas/test-sf-event-sqs-to-db-records.js

```diff
@@ -106,17 +151,21 @@ test('saveExecutionToDb() creates an execution item in Dynamo', async (t) => {
   }
 });
 
-test('saveExecutionToDb() does not throw an exception if storeExecutionFromCumulusMessage() throws an exception', async (t) => {
+test.serial('saveExecutionToDb() throws an exception if storeExecutionFromCumulusMessage() throws an exception', async (t) => {
   const { cumulusMessage } = t.context;
 
-  // Because state_machine is missing, generating this execution record will fail
-  delete cumulusMessage.cumulus_meta.state_machine;
+  const saveExecutionStub = sinon.stub(Execution.prototype, 'storeExecutionFromCumulusMessage')
+    .callsFake(() => {
+      throw new Error('error');
+    });
 
   try {
     await saveExecutionToDb(cumulusMessage);
-    t.pass();
+    t.fail('Exception should have been thrown');
```

Comment:
`t.throwsAsync` would work here, rather than `t.pass` and `t.fail`.

Reply:
ea55b31